src/cryptography/utils.py

```diff
@@ -10,6 +10,8 @@
 import sys
 import warnings
 
+from pkg_resources import parse_version
```

Comment:
Thanks @dstufft! @sigmavirus24 also suggested it in IRC so that looks like the path to go.